### PR TITLE
Submision of Latam W3 DAO

### PR DESCRIPTION
**Pull request by Juancamp1987**

### CODE REVIEW

File: CGPs/cgp-0076.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+# Proposal
+
+## Goal
+Bring to life next gen of dApps on Celo, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!),
+
+Celo has one of the most global communities in crypto. In order to further expand decentralization and the localization of the protocol in different regions and cultures, we must empower local communities to lead local growth. Decision power and initiatives must be shifted to folks on the ground who are the most equipped to collab and achieve a grassroots approach that will benefit the growth of Celo and its community.
+
+## Amount Requested
+$500k worth of CELO
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:33]: $550k to add the compensation

File: CGPs/cgp-0076.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+To spread awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish! 3), and help nurture the next cohort of great initiatives on Web3.
+Celo has one of the most global communities in crypto. 
+In order to further expand decentralization and the localization of the protocol in different regions and cultures, we would love to empower local communities to lead local growth.
+We believe that folks on the ground are the most equipped to do this and that a grassroots approach will benefit the growth of Celo and its community.
+
+
+# Proposal
+
+## Goal
+Bring to life next gen of dApps on Celo, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!),
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:28]: Bring to life next gen of dApps and partnerships to Celo by exploring #REFI use cases and opportunities, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!).

File: CGPs/cgp-0076.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+- Technical Support (encompasses tech development and business support)
+- Technical Support: 300 hours / Average professional hour at $80
+- Extras (e.g. back office/operations) $40,000
+
+#### Success Metrics:
+- 8 Web3 Dev Studio Participants across the region
+- 100% Project Conversion Rate to Celo
+- Transaction volume >$500M after 1 year
+
+
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:50]: Delete extra rows

File: CGPs/cgp-0076.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  - Content Creation/Moderation/Influencers	$45.000
+  - Promos/Swags/Paid Ads	$5.000
+
+- Success Metrics
+  - 2 Twitter Spaces per Week
+  - 2 Instagram Lives per week
+  - 4 Social media posts (Instagram, TikTok, Twitter) per Week
+  - Ongoing management of local communication channels
+
+
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:71]: Delete extra rows

File: CGPs/cgp-0076.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+- Forum posts with feedback request prior to initiatives
+- Forum posts with updates
+- Social media posts
+- Quarter reports
+- Open work on Discord channel
+Get to know our talent, and what we bring to the Celo Ecosystem!
+
+
+## Useful Links
+- Twitte/Instagram (@Celoorg.br @celo_esp)
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:107]: add "r" after Twitte

File: CGPs/cgp-0076.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+- 8 Web3 Dev Studio Participants across the region
+- 100% Project Conversion Rate to Celo
+- Transaction volume >$500M after 1 year
+
+
+### Collabs for Awareness, Education and Hackathons
+Help onboard new community members and developers, and guide them through the ecosystem, by collaborating with ecosystem founders and partners, doing Hackathons, meetups and other learning initiatives.
+
+#### Budget Ask: $120,000
+We are looking to accomplish at least 12 initiatives around the next year (6 in portuguese and 6 in spanish). Ecosystem partners must submit their proposals for such and the total budget will be allocated to them. We will co-organize the initiative together and report to the community.
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:55]: We are looking to accomplish at least 12 initiatives around the next year (6 in Portuguese and 6 in Spanish). Ecosystem partners (i.e. Founders on Celo) must submit their proposals for such and budget will be allocated after open calls for discussion. We may co-organize the initiatives together and report to the community.

File: CGPs/cgp-0076.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+Celo has one of the most global communities in crypto. In order to further expand decentralization and the localization of the protocol in different regions and cultures, we must empower local communities to lead local growth. Decision power and initiatives must be shifted to folks on the ground who are the most equipped to collab and achieve a grassroots approach that will benefit the growth of Celo and its community.
+
+## Amount Requested
+$500k worth of CELO
+
+## Areas of Focus
+
+### Accelerating Growth: Web3 Venture Builder (Solution Design + Project Development and Shipping + Capital)**
+Create a Web3 Venture Studio to launch the next gen of Web3 initiatives by: (i) helping expressive Web2 companies launch on Celo and (ii) nurturing native web3 initiatives.
```
Comment (MilaRioja) [inline CGPs/cgp-0076.md:38]: add: Founders on Celo community may play a joint role in this initiative: either by providing services, collaborating and/or other formats that we feel may further advance the Celo mission.

**Review by willkraft — APPROVED** [review]

**Review by ericnakagawa — APPROVED** [review]
Thanks for the work Juan and team!

**Pull request merged**